lib/zoho_auth_client.rb

```diff
@@ -0,0 +1,40 @@
+class ZohoAuthClient
+  ACCESS_TOKEN_REQUEST_ENDPOINT = 'https://accounts.zoho.com/oauth/v2/token'.freeze
+
+  CLIENT_ID = ArchiveConfig.ZOHO_CLIENT_ID
+  CLIENT_SECRET = ArchiveConfig.ZOHO_CLIENT_SECRET
+  ZOHO_REFRESH_TOKEN = ArchiveConfig.ZOHO_REFRESH_TOKEN
+
+  # TODO: Can we get rid of this? We don't redirect, but it has to match
+  # whatever was filled in when creating the original grant_code
```

Comment:
And we have to have it, maybe we should change it to the homepage, just because it's weird to have it be `/support` when we're using it for both the Support and PAC forms. (Not that it makes a big difference considering it's not being used for anything, but...)

Reply:
Actually... this should probably go in the config file as `ZOHO_REDIRECT_URI`.